src/exceptions/oauth.exception.ts

```diff
@@ -123,4 +125,8 @@ export class OAuthException extends Error {
       HttpStatus.INTERNAL_SERVER_ERROR,
     );
   }
+
```

Comment:
the getter is kinda stupid, I just added it because of the tsconfig setting for no unused fields